sound/soc/sof/intel/cnl.c

```diff
@@ -150,14 +150,36 @@ static void cnl_ipc_dsp_done(struct snd_sof_dev *sdev)
 				CNL_DSP_REG_HIPCCTL_DONE);
 }
 
+static bool cnl_is_ipc_compact(struct snd_sof_ipc_msg *msg)
+{
+	/* the compact IPC subset might be extended later */
+	if (msg->header == (SOF_IPC_GLB_PM_MSG | SOF_IPC_PM_GATE))
+		return false;
+	else
+		return true;
```

Comment:
@keyonjie is the logic inverted here? Should you return true if the if() is satisfied?

Reply:
you are right, thanks for spotting it.

Reply:
now updated.